src/openforms/appointments/contrib/jcc/tests/test_admin.py

```diff
@@ -0,0 +1,104 @@
+from unittest.mock import patch
+
+from django.urls import reverse
+
+import requests
+import requests_mock
+from django_webtest import WebTest
+
+from openforms.accounts.tests.factories import SuperUserFactory
+from openforms.appointments.models import AppointmentsConfig
+from openforms.utils.tests.logging import disable_logging
+
+from .utils import MockConfigMixin
+
+
+@disable_logging()
+class ApointmentConfigAdminTests(MockConfigMixin, WebTest):
+    @requests_mock.Mocker()
+    def test_admin_while_service_is_down(self, m):
+        m.register_uri(
+            requests_mock.ANY, requests_mock.ANY, exc=requests.RequestException
+        )
+
+        config = AppointmentsConfig.get_solo()
+        assert isinstance(config, AppointmentsConfig)
+        config.plugin = "jcc"
+        config.limit_to_location = "test"
+        config.save()
+
+        superuser = SuperUserFactory.create()
+        response = self.app.get(
+            reverse(
+                "admin:appointments_appointmentsconfig_change",
+                args=(AppointmentsConfig.singleton_instance_id,),
+            ),
+            user=superuser,
+        )
+        self.assertEqual(response.status_code, 200)
+
+        plugin = response.context["adminform"].form["plugin"]
+        self.assertEqual(plugin.value(), config.plugin)
+
+        limit_to_location = response.context["adminform"].form["limit_to_location"]
+        self.assertEqual(limit_to_location.value(), config.limit_to_location)
```

Comment:
preferably you use `response.forms["..."]` which will give you a webtest response

e.g. assertion:

```py
self.assertEqual(form["plugin"], "jcc")
```